README.md

```diff
@@ -233,6 +233,23 @@ This currently works automatically for the `target/kubernetes/openshift.yml` fil
 
 Note that it is usually a good idea to set `-Dts.image-overrides` to a _full_ path, because Maven changes the current working directory when running tests.
 
+### Native image
+
+The test suite contains a Maven profile configuring required system properties for native image build. The profile is
+activated by specifying the `native` Quarkus profile (`-Dquarkus.profile=native`). This profile does not specify the 
+container runtime for Quarkus builder image, so by default, native build will use local GraalVM. The following command 
```

Comment:
"container runtime for Quarkus builder image" -- the term "builder image" is overloaded, I'd use "container runtime for Quarkus native build".